template-deliverables.md

```diff
@@ -0,0 +1,145 @@
+It provides a lightweight introduction to the purpose of this documentation and explains how to fill in each section of the document.
+
+In essence, this guide supports the key decisions that the template user might need to make during the writing process.
+
+This guide might also optionally include a checklist of the required components for this type of document.
+It might also include a list of helpful resources or examples that were consulted when the template was created.
+
+
+### Content and formatting guidelines
+
+The template guide should follow the same basic structure:
+
+- **Template guide title** - The type of document this template is for and the words "guide." For example, "Tutorial guide" or "README guide." This title should be formatted with the heading 1 weight.
+- **Introductory comment** - Include an embedded writing tip at the start that tells users they should first read this guide and the before-you-start guide before they fill in the template. This comment could also mention how to get the raw template file. Put introductory comments in {curly brackets}.
+- **Why do I need this type of document?** - Provide some of the information about the purpose of this type of document and how it helps your documentation project. The header for this section should be formatted with the heading 2 weight.
+- **Content of this template** - This header marks the beginning of the part of the guide that explains how to fill in each section of the guide. It should be formatted with the heading 2 weight.
+- **About the {insert section name here} section** - Next, include each section as they appear in the template with the heading 3 weight. For example, if your template includes a "Before you start" section, this heading should say "About the before your start section." And then it should provide guidance about what kind of content goes in that section and why that section might be included in the final document. If the section is optional, indicate why some documents could benefit from that section or why it might be left out. If the template user needs to make a decision about the content in that section, provide guidance about what should go into that section.
+- **Checklist (optional)** - If it makes sense to include a checklist of the typical content required for this type of document, include that here.
+- **Additional resources (optional)** - If you’d like to point the user to any helpful guides for deeper study or if you want to acknowledge any resources that inspired you while you wrote this template, link to or cite those resources here.
+
+### Frequently asked questions
+
+Q: No questions yet.
+
+
+## Before you start guide (optional)
+
+Example filename: before-you-start-abc-template.md
+
+This optional guide is designed to help template users understand some of the key research, brainstorming, or pre-writing steps that they might need to do before adding this type of document to a guide.

```

Comment:
This all seems to be covered by the guide file. Even more confusingly, this file is referred to as a "guide" in several places. If we keep both this file and the guide file with their current names, we should avoid referring to this file as a guide to avoid confusion.

Reply:
I suggest that we expand our target audience here to:
* Docset owner, building the information architecture for a project. This person may be a technical writer.
* Curious and engaged author, who wants to know more than "just follow the steps in the guide".
* Not your "just do write a doc" author.
Note: We've moved back and forth on deciding whether sections fit in the guide or theory/primer doc.

Reply:
See my reply to your previous comment.

Reply:
I've revised this section based on what we agreed in last week's general meeting. Resolved: We discussed this at length at last week's general meeting and the group consensus there was to call this the "deep dive" document. It basically contains miscellaneous information or guidance that is too comprehensive for the "guide" document. Based on whatever the each unique template needs, this document could include research, deep theories, pre-writing information, and/or any other material that users need to know when implementing this topic. Template writers are encouraged to point/link readers to that document in their guide to encourage users to read it in more depth.